pkg/apis/kops/componentconfig.go

```diff
@@ -1088,8 +1088,8 @@ type CertManagerConfig struct {
 	HostedZoneIDs []string `json:"hostedZoneIDs,omitempty"`
 }
 
-// AWSLoadBalancerControllerConfig determines the AWS LB controller configuration.
-type AWSLoadBalancerControllerConfig struct {
+// LoadBalancerControllerSpec determines the AWS LB controller configuration.
+type LoadBalancerControllerSpec struct {
```

Comment:
I agree with the Config -> Spec bit, not sure we want to drop the AWS bit.  If/when we add the GCE LoadBalancerController, won't we readd the AWS prefix?

I think we tend to end up with the path to the type in the type name, not the field name (e.g. PodSpec and NodeSpec).

Reply:
We could always change it back at that point. It doesn't affect the API.